Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -146,6 +173,32 @@ of removing unused segments after release of internal resources.
     - **tier**: Consolidate based on segment byte size and live document count
       as dictated by the customization attributes.
 
+### `consolidationPolicy` properties for `tier` type
+
+  - **min\_segments** (_optional_; type: `integer`; default: `1`)
```

Comment:
The backslash escape for the underscore shouldn't be necessary here. It is not used above for the `bytes_accum` property, and that one is in for quite some time and doesn't seem to cause any trouble:
https://docs.arangodb.com/3.4/Manual/Views/ArangoSearch/DetailedOverview.html#view-properties

```suggestion
  - **min_segments** (_optional_; type: `integer`; default: `1`)
```